enc_dec.py

```diff
@@ -0,0 +1,249 @@
+import sys
+import bech32m
+from colorama import init, Fore, Style
+
+
+init(autoreset=True)  # Initializes colorama
+CHARSET = "qpzry9x8gf2tvdw0s3jn54khce6mua7l"
+
+
+def validate_hrp(hrp):
+    """
+    validates the entered human-readable part for encoding/decoding a string
+    :param hrp: valid ascii string
+    :return: True if hrp meets the criteria specified else False
+    """
+
+    if hrp == "":
+        print("HRP cannot be null/empty")
+        return False
+
+    if len(hrp) > 83:
+        print("Invalid Length of HRP. Enter Valid HRP string of length between 1 and 83")
+        return False
+
+    for x in hrp:
+        if ord(x) < 33 or ord(x) > 126:
+            print("Invalid character in HRP. Character should have ASCII Value between 33 and 126(both inclusive")
+            return False
+        else:
+            return True
+
+
+def validate_str_for_encoding(hrp, input_str, input_format):
+    """
+    validates a string for encoding and return validated string
+    :param hrp: string to specify Human-readable part
+    :param input_str: string to be encoded
+    :param input_format: input format of string to be encoded
+    :return: raw bytes of validated string
```

Comment:
issue (documentation): 4/7 return statements return either False or None, not raw bytes of validated string.